cli/src/semgrep/commands/ci.py

```diff
@@ -250,6 +252,24 @@ def ci(
     else:  # impossible state… until we break the code above
         raise RuntimeError("The token and/or config are misconfigured")
 
+    if beta_testing_secrets:
+        if supply_chain:
+            # Not logged in and no explicit config
+            logger.info("Cannot use both `--supply-chain` and `--beta-testing-secrets`")
+            sys.exit(FATAL_EXIT_CODE)
```

Comment:
This is 100% what we talked about doing, so sorry for being inconsistent, but I had a random thought today - what is the harm to letting people use both of these flags and in that case returning supply chain and secrets findings (but not code findings)?